pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -115,6 +115,19 @@ rec {
         ]
       ];
 
+      defaultHardeningFlags = [
+        "fortify" "stackprotector" "pic" "strictoverflow" "format" "relro" "bindnow"
+      ];
+
+      hardeningDisable = lib.toList (attrs.hardeningDisable or [ ]);
```

Comment:
These are already bound above :)

Reply:
The reason for this line (and the similar line below) is because I wanted to support the (at least historical) use case of passing a string (e.g. `"all"`) for these arguments, but to simplify logic down below, I wanted to coerce that value to a list.

At some point after I initially submitted this PR, it looks like we made the interface for `mkDerivation` such that `hardeningEnable` & `hardeningDisable` strictly *require* a list, rather than accepting a list *or* string. In light of that, yeah, these lines needn't exist :).